elliott/elliottlib/bzutil.py

```
@@ -597,6 +661,89 @@ def attach_bugs(self, bugids: List, advisory_id: int = 0, advisory_obj: Erratum
+        :param private: Whether the comment should be private
+        :param noop: If True, only log what would be done
+        :return: True if comment was added, False if skipped
+        """
+        # Check if we've already commented
+        try:
+            existing_comments = self.get_comments(bugid)
+            for existing_comment in existing_comments:
+                if marker in existing_comment:
+                    logger.info(f"Skipping comment on {bugid} - marker '{marker}' already exists")
+                    return False
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)